test/bundler/esbuild/importstar.test.ts

```diff
@@ -561,6 +561,7 @@ describe("bundler", () => {
     format: "iife",
   });
   itBundled("importstar/ExportSelfIIFEWithName", {
+    todo: true,
```

Comment:
i'm scoping out support for IIFE in this PR

Reply:
reasonable.